src/render/mesh/trimesh.rs

```diff
@@ -0,0 +1,58 @@
+use bevy::prelude::*;
+use bevy::render::pipeline::PrimitiveTopology;
+use bevy::render::mesh::{Indices, VertexAttributeValues};
+use crate::prelude::*;
+
+// TODO: Fix this
+// The first and last point should be connected to complete the polygon. This leaves one side
+// of a quad (for example) without a side.
+//  _____
+//  x  /|
+//  x / |
+//  ‾‾‾‾
+
+#[cfg(feature = "dim3")]
+pub fn wire_trimesh(trimesh: &TriMesh) -> Mesh {
+    let mut mesh = Mesh::new(PrimitiveTopology::LineList);
```

Comment:
I think you can fix this by using:
```suggestion
    let mut mesh = Mesh::new(PrimitiveTopology::LineStrip);
```

![image](https://user-images.githubusercontent.com/5142292/148357814-e07ccf34-146e-4b87-9274-3e2cf289cc78.png)

Reply:
This might actually be adding too many lines?

Reply:
It's been a while since i've looked at this but, if i remember correctly the code to generate the timesh mesh was basically just taken from the current debug renderer.